src/test/run-make/codegen-options-parsing/Makefile

```diff
@@ -22,3 +22,10 @@ all:
 	$(RUSTC) -C lto=foo dummy.rs 2>&1 | \
 		grep 'codegen option `lto` takes no value'
 	$(RUSTC) -C lto dummy.rs
+
+	# Should contain gc-sections...
+	$(RUSTC) -Z print-link-args dummy.rs 2>&1 | \
+		grep -e '--gc-sections\|-dead_strip'
```

Comment:
I didn't grep on any other value because only 2 are defined in [gc_sections()](https://github.com/rust-lang/rust/blob/master/src/librustc_trans/back/linker.rs#L117)